bundle/pom.xml

```diff
@@ -461,11 +479,16 @@
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>org.apache.sling</groupId>
-            <artifactId>org.apache.sling.commons.scheduler</artifactId>
-            <version>2.3.4</version>
-            <scope>provided</scope>
+            <groupId>com.adobe.aem</groupId>
+            <artifactId>aem-api</artifactId>
+            <version>6.0.0.1</version>
         </dependency>
+        <dependency>
```

Comment:
@justinedelson let me double check this; i dont think this PR should be dependent on 6.0...